packages/wrangler/src/deploy/index.ts

```diff
@@ -270,14 +277,43 @@ export async function deployHandler(
 	}
 
 	if (
-		(args.legacyAssets || config.legacy_assets) &&
+		(args.legacyAssets ||
+			config.legacy_assets ||
+			args.experimentalAssets ||
+			config.experimental_assets) &&
 		(args.site || config.site)
 	) {
 		throw new UserError(
 			"Cannot use Assets and Workers Sites in the same Worker."
 		);
 	}
 
+	if (
+		args.experimentalAssets &&
+		!existsSync(path.resolve(process.cwd(), args.experimentalAssets))
```

Comment:
Are asset directory paths relative to the cwd or to the wrangler.toml?

Reply:
The legacy assets stuff appears to use cwd if `--assets` is defined in the command line, and the wrangler.toml directory if using the `assets` property in the config file.

Reply:
See `getAssetPaths()` helper for more info.

Reply:
indeed, this was an open question that I mean to bring up today. I'll have a look, at `getAssetsPath`

Reply:
fixed in https://github.com/cloudflare/workers-sdk/pull/6312/commits/7b204ca6084bfc276ce67200198aff951661430a#diff-99329bc31323333c32855181ed3b738e17d682a5f16c825ec49be45d30e54082R8